frontend/components/AddHostsModal/PlatformWrapper/PlatformWrapper.tsx

```diff
@@ -54,6 +60,19 @@ interface IPlatformWrapperProps {
   onCancel: () => void;
 }
 
+const CHROME_INFO = {
```

Comment:
For clarity and since we'll be referring to it in quite a few places, can we use "ChromeOS" instead of "Chrome" to distinguish what we're talking about from the browser? So this could be `CHROME_OS_INFO` perhaps